modules/preview-react/menu/stories/Menu.stories.mdx

```diff
@@ -27,8 +26,15 @@ yarn add @workday/canvas-kit-preview-react
 
 ### Basic Example
 
+`Menu` is typically triggered by an action such as pressing a button. Here's an example of how you
+might implement that behavior using a [`Popup`](/components/popups/popup/).
+
 <ExampleCodeBlock code={Basic} />
 
+`Menu` will automatically assign focus to itself when it appears provided you set the `isOpen` prop
+correctly, so you do **not** need to use the `useInitialFocus` `Popup` hook. You **will**, however,
+need to use `useReturnFocus` to return focus back to the triggering button after closing the `Menu`.
```

Comment:
I added this note about `useInitialFocus` not being necessary since Menu already [gives itself focus](https://github.com/Workday/canvas-kit/blob/master/modules/preview-react/menu/lib/Menu.tsx#L92) when it's opened. I tried not setting the `isOpen` prop and using `useInitialFocus` instead to set focus on the Menu, but that was causing the entire page to scroll to the top after clicking the button to activate the Menu.

Maybe an issue with how focus is being managed internally in Menu, not sure. I decided to continue relying on the Menu to focus itself.